### DISCUSSION TITLE
Settings organ deletion

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1027] [Developers]
I noticed when it's asked confirmation Y/N when deleting organs , not when deleting package organs below. Is this by design ?

Question: is it possible to enable shift/ctrl when deleting organs so to be able to select more than one at a time ?

**Comment by oleg68** [comment]
Now it is not possible. Please add youw wishes to the requirements to the new design of the Organs tab #955
